index.html

```diff
@@ -0,0 +1,72 @@
+</head>
+<body>
+    <div class="nav-container">
+        <div class="wrapper">
+            <div  class="logo">
+                <img src="images/mobile/my-logo.png" alt="image"/>
+            </div>
+            <div class="toggle-div">
+                <img src="images/mobile/Enabled.png" alt="image"/>
+            </div>
+            <!-- <nav>
+                <ul>
+                    <li class="portfolio"><a href="#">Portfolio</a></li>
+                    <li class="about"><a href="#">About</a></li>
+                    <li class="contact"><a href="#">Contact</a></li>
+                </ul>
+            </nav> -->
+        </div>
+
+        <div class="hero">
+           
+          
+         
+           
+            <img src="images/mobile/Rectangle-6.png" class="rectangle-6" alt="image"/>
+            <div class="eclipse-11"><img src="images/mobile/Ellipse-11.png" alt="image"/></div>
+            <div class="eclipse-8"><img src="images/mobile/Ellipse-8.png" alt="image"/></div>
+            <div class="rectangle-7"><img src="images/mobile/Rectangle-7.png" alt="image"/></div>
+            <div class="eclipse-5"><img src="images/mobile/Ellipse-5.png" alt="image"/></div>
+             <div class="rectangle-4"><img src="images/mobile/Rectangle-4.png" alt="image"/></div>
+           <div class="eclipse-3"><img src="images/mobile/Ellipse-3.png" alt="image"/></div>
+            <div class="eclipse-14"><img src="images/mobile/Ellipse-14.png" alt="image"/></div>
+            
+            <div class="text">
+                <h1>I’m Alexandra <br/> Glad to see <br/> you!</h1>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+
+            <div class="let-chat">
+                <h6>LET’S CONNECT</h6>
+                <p>
+                    <img src="images/mobile/twitter.png" alt="image"/>
+                    <img src="images/mobile/linkedin.png" alt="image"/>
+                    <img src="images/mobile/github.png" alt="image"/>
+                    <img src="images/mobile/m.png" alt="image"/>
+                    <img src="images/mobile/fingers.png" alt="image"/>
+                </p>
+            </div>
+           
+           <div class="eclipse-12"><img src="images/mobile/Ellipse-12.png" alt="image"/></div>
+            <div class="rectangle-5"><img src="images/mobile/Rectangle-5.png" alt="image"/></div>
+            <div class="eclipse-4"><img src="images/mobile/Ellipse-4.png" alt="image"/></div>
+            <div class="eclipse-15"><img src="images/mobile/Ellipse-15.png" alt="image"/></div> 
+            <div class="eclipse-13"><img src="images/mobile/Ellipse-13.png" alt="image"/></div> 
+        
+        </div>
+    </div>
```

Comment:
- [ ] It will be good to wrap this section using a `section` tag instead of a div, to follow HTML/CSS best practice of using semantic tags.

Reply:
I have wrapped this section using a section tag